gemfire-core/src/main/java/com/gemstone/gemfire/internal/cache/GemFireCacheImpl.java

```diff
@@ -602,11 +603,24 @@ public void removeRegionFromOldEntryMap(String regionPath) {
     }
   }
 
+  public long getOldEntryRemovalPerid() {
+    return OLD_ENTRIES_CLEANER_TIME_INTERVAL;
+  }
```

Comment:
Have mentioned to Sachin too. Do you need time interval to be so high at 60 secs now that the race condition has been taken care of with the updateInProgress flag? More like 5 secs should do. We can look at the method itself and try to make it as efficient as possible to minimize the overhead. Going forward lets discuss again if a guava weakKeys/Values map can be used here (I also forget why a map was required at all).